docs/readmes/lte/sessiond.md

```diff
@@ -22,7 +22,7 @@ SessionD interfaces with three distinct external components.
  reports usage accordingly.
  
 ### Internal Architecture 
-![Internal Architecture Diagram](SessionD_Architecture.png)
+![Internal Architecture Diagram](https://github.com/magma/magma/blob/master/docs/readmes/assets/SessionD_Architecture.png)
```

Comment:
It's better to render the relative path even though it breaks local markdown viewers. If this image undergoes revisions, we want previous doc versions to show the correct image for the version that they document. Not a big fan of linking out either when the asset is bundled right there anyways